mailet/standard/src/main/java/org/apache/james/transport/mailets/ReplaceContent.java

```diff
@@ -114,7 +114,7 @@ public void init() throws MailetException {
     }
 
     private boolean isDebug() {
-        return Integer.valueOf(getInitParameter("debug", "0")) == 1;
+        return getInitParameter("debug", false);
```

Comment:
That, is an API change. You will break peoples ReplaceContent debug configuration as the `1` value (which I agree is not explicit at all) will be broken.